docs/content/docs/engines/hive.md

```diff
@@ -137,7 +131,7 @@ SELECT * FROM test_table;
 Run the following Hive SQL in Hive CLI to access the created table.
 
 ```sql
--- Assume that flink-table-store-hive-connector-{{< version >}}.jar is already in auxlib directory.
+-- Assume that flink-table-store-hive-connector-2.3-{{< version >}}.jar is already in auxlib directory.
```

Comment:
`2.3` -> `<hive-version>`. There are other appearance of `flink-table-store-hive-connector-{{< version >}}.jar` in the document. Please replace them.